README.md

```diff
@@ -864,6 +864,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Censys.io](https://censys.io/api) | Search engine for Internet connected host and devices | `apiKey` | Yes | No |
 | [CRXcavator](https://crxcavator.io/apidocs) | Chrome extension risk scoring | `apiKey` | Yes | Unknown |
+| [Escape](https://github.com/polarspetroll/EscapeAPI)| an API for escaping different kind of queries | No | Yes | No 
```

Comment:
- Put a blank space between the link and `|` character
- Start the description with a capital letter
- And at the end of the line put a `|`

```suggestion
| [Escape](https://github.com/polarspetroll/EscapeAPI) | An API for escaping different kind of queries | No | Yes | No |
```